cmd/openshift-tests/images.go

```diff
@@ -207,6 +207,10 @@ func pulledInvalidImages(fromRepository string) ginkgo.JUnitForEventsFunc {
 		// TODO: will not work for a disconnected test environment and should be emulated by launching
 		//   an authenticated registry in a pod on cluster
 		"registry.redhat.io/ubi8/nodejs-14:latest",
+
+		// TODO: temporarily allow pulling to facilitate rebase
+		//  https://bugzilla.redhat.com/show_bug.cgi?id=2093339
```

Comment:
I'd prefer to skip the entire test in https://github.com/openshift/origin/blob/master/test/extended/util/annotate/rules.go rather than allowing this error here. The test will fail in disconnected installation and that's why skipping it entirely is better than this. Also this is a net-new test with an urgent BZ to track it for 4.11.
/hold